src/test/kotlin/org/rust/lang/core/resolve/RsProcMacroResolveTest.kt

```diff
@@ -112,19 +112,33 @@ class RsProcMacroResolveTest : RsResolveTestBase() {
         struct S;
     """)
 
-    // FIXME
-    fun `test resolve attr proc macro from macro call with full path`() = expect<IllegalStateException> {
-        stubOnlyResolve("""
+    @MockEdition(Edition.EDITION_2018)
+    fun `test resolve attr proc macro from macro call with full path`() = stubOnlyResolve("""
         //- dep-proc-macro/lib.rs
             #[proc_macro_attribute]
             pub fn example_proc_macro(attr: TokenStream, item: TokenStream) -> TokenStream { item }
         //- lib.rs
-            use dep_proc_macro
             #[dep_proc_macro::example_proc_macro]
                                 //^ dep-proc-macro/lib.rs
             struct S;
     """)
-    }
+
+    @MockEdition(Edition.EDITION_2018)
```

Comment:
Lets remove `@MockEdition` from individual methods since we have `@MockEdition` on the whole class